client-runtime/src/main/java/com/microsoft/rest/v2/annotations/Doc_Http_Verb_Annotation_Value_URI_Template.java

```diff
@@ -36,7 +36,7 @@
  *  The name of these parameters are predefined, if they are present then they must be present exactly in places where they are suppose to present in a valid URI.
  *
  *  Value of {protocol} is derived from {@link EndpointProtocol} annotation. Refer documentation of {@link EndpointProtocol} for more details.
- *  Value of {host} is derived from {@link EndpointHost} annotation. Refer documentation of {@link EndpointHost} for more details.
+ *  Value of {host} is derived from {@link Host} annotation. Refer documentation of {@link Host} for more details.
```

Comment:
Why did you change this from EndpointHost to Host when the other two values have an "Endpoint" prefix?